lib/stack_master/validator.rb

```diff
@@ -13,6 +13,7 @@ def perform
     rescue Aws::CloudFormation::Errors::ValidationError => e
       $stderr.puts "Validation Failed"
       $stderr.puts e.message
+      false
```

Comment:
This change makes the return value the instance of the command instead of the return value of perform.

```
    module ClassMethods
      def perform(*args)
        new(*args).tap { |command| command.perform }
      end
    end
```